### PR TITLE
Add an entrypoint for running the jobrunner stats extraction

### PR DESCRIPTION
**Pull request by rebkwok**
Uses the osrelease config to run the jobrunner stats extraction script (as described [here](https://bennettinstitute-team-manual.pages.dev/tech-team/extract-job-stats/)) and push the extracted file to a github repo.

### CODE REVIEW

File: publisher/config.py [hunk 1]
```diff
@@ -162,5 +164,29 @@ def load_config(options, release_dir, env=os.environ):
                 files = git_files(release_dir)
             else:
                 sys.exit("No files provided to release")
+    return files, config
+
+
+def load_jobrunner_stats_config(cfg):
+    username = get_username(cfg)
+    config = {
+        "backend_token": cfg.get("BACKEND_TOKEN"),
```
Comment (bloodearnest) [inline publisher/config.py:173]: I don't think you need the backend_token for the extraction code?

No big deal though.

File: publisher/config.py [hunk 2]
```diff
@@ -121,38 +131,30 @@ def load_config(options, release_dir, env=os.environ):
...
 
-    if options.new_publish:
+    if getattr(options, "new_publish", False):
         # must provide files in new publish
         if not files:
             sys.exit("No files provided to release")
     else:
-        # deprecated github publishing
-        config["study_repo_url"] = manifest["repo"]
+        # deprecated github publishing or jobrunner-stats extraction
```
Comment (bloodearnest) [inline publisher/config.py:157]: think the 'or ..' part of this comment is defunct since we have `load_jobrunner_stats_config`?

**Review by bloodearnest — APPROVED** [review]
LGTM, couple of comments if you want to

File: publisher/config.py (hunk 2 above)
Comment (rebkwok) [reply publisher/config.py:157]: oh yes, forgot to remove

**Comment by sebbacon** [comment]
Looks good, I suppose we need to update [the docs](https://bennettinstitute-team-manual.pages.dev/tech-team/extract-job-stats/) after we merge this..?

**Comment by bloodearnest** [comment]
looks great, thanks

**Pull request merged**